README.md

```diff
@@ -56,5 +56,18 @@ Simple queueing service built for Internet Computer
 - Send & read messages simultaneously
 - Once processed only
 
+---------
 
+#### <b>DFX Commands for local testing</b>
 
+```
+$ dfx deploy --mode=reinstall --argument '(opt record {principal "<principal-id>"; null})' icsqs  
```

Comment:
can you replace this principal id with an actual value taken from dfx identity get-principal ?